posts/postRoutes.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const db = require('../data/db.js');
+
+const router = express.Router();
+
+router.post('/', (req, res) => {
+  const { title, contents } = req.body;
+  const newPost = { title, contents };
+  if (!title || !contents) {
```

Comment:
Great use of an if statement but maybe change it to && instead of || that way the user can't skip either input